README.md

```diff
@@ -1,2 +1,62 @@
-# weather-app
-The application uses an API weather service to find the info to different parts across the world.
+<h3 align="center">Weather app - Javascript</h3>
+
+
+<p align="center">Weather app in JavaScript is a practice project, which includes the API service to see the weather to more than 200,000 cities around the world. We used Javascript, HTML5 and CSS3, to implement it.</p>
+
+![screenshot](/screenshots/Screenshot_1.png)
+
+## Built With
+
+- JavaScript
+- HTML5
+- Bootstrap
+- CSS3
+- [openweathermap.org](https://openweathermap.org/current)
+
+### Prerequisites
+
+- Text Editor (VSCode is suggested.)
+
+### Setup
+
+- Install [git](https://git-scm.com/downloads)
+- Open Terminal
+- Change directory to folder to download repository
+- Use `cd <file-path>`
+- Run `git clone https://github.com/vichuge/weather-app.git`
+- open the folder
+- double click index.html
+- Here comes the website on the local machine.
```

Comment:
The setup instruction in the Readme is not complete: you should add instructions on how the user can build for deployment, run a local server, etc.